pages/products.html

```diff
@@ -0,0 +1,308 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/products.css" />
+    <link rel="stylesheet" href="../css/styles.css"/>
+    <link
+      rel="stylesheet"
+      href="https://dreamui-demo.netlify.app/dreamui-v1.css"
+    />
```

Comment:
you can import all these links in one main file and then import that in your css file, so that you don't have to repeat this in every html file

Reply:
For now, I will keep it as it is as there are only two CSS files(component library link and page.css) that I have to import in html, but will consider this in future while refactoring code